app/src/main/java/io/multy/ui/activities/SplashActivity.java

```diff
@@ -114,12 +116,20 @@ private void showUpdateDialog() {
     }
 
     private void showMainActivity() {
-//        Animation leave = AnimationUtils.loadAnimation(this, R.anim.splash_leave);
-//        leave.setInterpolator(new FastOutLinearInInterpolator());
-//        leave.setFillAfter(true);
-//        icon.startAnimation(leave);
-        finish();
-        overridePendingTransition(R.anim.alpha_250, R.anim.splash_leave);
-        startActivity(new Intent(SplashActivity.this, MainActivity.class));
+        Thread background = new Thread() {
```

Comment:
This looks very bad. Can we avoid it?

Reply:
No. It remove lags from activity switching.